tests.js

```diff
@@ -542,6 +542,18 @@ describe('helpers: isEmpty', () => {
 	});
 });
 
+describe('helpers: isFraction', () => {
+	it('should identify string fractions as fractions', () => {
+		expect(helpers.isFraction('4/3')).to.equal(true);
+	});
+
+	it('should not identify any other types', () => {
+		expect(helpers.isFraction(4/3)).to.equal(false);
```

Comment:
@jmtaylor You can write your assertions in a little cleaner way for boolean responses. You can simply write: `expect(value).to.be.true` or `expect(value).to.be.false`.